src/types.ts

```diff
@@ -0,0 +1,56 @@
+export interface Props {
+    data: QueriedData
+}
+interface QueriedData {
+    count: number,
+    next: string,
+    previous: string | null,
+    results: BeneficiaryData[]
+}
+
+export interface BeneficiaryData {
+    id:                 number;
+    first_name:         string;
+    last_name:          string;
+    other_name:         null;
+    gender:             string;
+    profile_photo:      string | null;
+    phone_number:       string;
+    email:              string;
+    beneficiary_ID:     string;
+    date_of_birth:      Date;
+    marital_status:     string;
+    name_of_spouse:     string | null;
+    number_of_children: number | null;
+    number_of_siblings: number | null;
+    education_level:    string | null;
+    created:            Date;
+    agent_ID:           AgentID;
+    parent_details:     ParentDetails;
+}
+
+export interface AgentID {
```

Comment:
why are we calling this `AgentID` instead of just `Agent`?

Reply:
I did not think of renaming that, it makes sense to call it Agent.